Objects/unicodeobject.c

```diff
@@ -15120,7 +15120,7 @@ unicode_subtype_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
 
 PyDoc_STRVAR(unicode_doc,
 "str(object='') -> str\n\
-str(bytes_or_buffer[, encoding[, errors]]) -> str\n\
+str(object, encoding=\"utf-8\", errors=\"strict\") -> str\n\
```

Comment:
Best to use single quotes to be consistent with the line above.

Since you add the defaults to the signature, I think you can remove them from the end of the text below.